app/controllers/admin/slides_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Admin::SlidesController < Admin::ApplicationController
+  def index
+    ransack_params = params[:q]
+    @q = Slide.search(ransack_params)
```

Comment:
__[reek]__
* [UncommunicativeVariableName] has the variable name '@q' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Variable-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->